extensions/lima/src/extension.ts

```
@@ -101,19 +117,29 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   const socketPath = path.resolve(limaHome ?? '', instanceName + '/sock/' + socketName);
   const configPath = path.resolve(limaHome ?? '', instanceName + '/copied-from-guest/kubeconfig.yaml');
 
+  const installedLima = await getLimaInstallation();
+  const version: string | undefined = installedLima?.version;
+
   let provider;
```

Biome (2.4.7)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)